src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/naming/localfinalvariablename/Example1.java

```diff
@@ -6,5 +6,8 @@
 </module>
 */
 
+package com.puppycrawl.tools.checkstyle.checks.naming.localfinalvariablename;
+
 // xdoc section -- start
+class Example1{}
 // xdoc section -- end
```

Comment:
Please do us a favor copy Example2 class content to Example1.
And add Example1 to template file.
Looks like we lost Example1 usage at all

Reply:
done